src/metatrain/utils/additive/zbl.py

```diff
@@ -0,0 +1,309 @@
+from typing import Dict, List, Optional
+
+import metatensor.torch
+import torch
+from ase.data import covalent_radii
+from metatensor.torch import Labels, TensorBlock, TensorMap
+from metatensor.torch.atomistic import ModelOutput, System
+
+from ..data import DatasetInfo
+from ..jsonschema import validate
+
+
+class ZBL(torch.nn.Module):
+    """A simple model for short-range repulsive interactions.
+
+    :param model_hypers: A dictionary of model hyperparameters. This contains the
+        "inner_cutoff" and "outer_cutoff" keys, which are the inner and outer cutoffs
+        for the ZBL potential.
+    :param dataset_info: An object containing information about the dataset, including
+        target quantities and atomic types.
+    """
+
+    def __init__(self, model_hypers: Dict, dataset_info: DatasetInfo):
+        super().__init__()
+
+        # `model_hypers` should contain an "inner_cutoff" key
+        # and an "outer_cutoff" key
+        validate(
+            instance=model_hypers,
+            schema={
+                "type": "object",
+                "properties": {
+                    "inner_cutoff": {"type": "number"},
+                    "outer_cutoff": {"type": "number"},
+                },
+                "required": ["inner_cutoff", "outer_cutoff"],
+                "additionalProperties": False,
+            },
+        )
+
+        # Check capabilities
+        if dataset_info.length_unit != "angstrom":
+            raise ValueError(
+                "ZBL only supports angstrom units, but a "
+                f"{dataset_info.length_unit} unit was provided."
```

Comment:
I think we can do unit conversion in the end no? Should be fairly trivial. The values are there in metatensor atomistic.

Reply:
I don't know... I think it's a bit tricky and I thought the best way for now would be to enforce Angstroms and eV. When would you do the unit conversion (both at training time and at prediction time)? And what if the user doesn't specify any units?

I wonder if @Luthaf has any ideas for this

Reply:
Yeah, I agree that it might be a bit painful, but since we have `units` in our config I think we should support them. I would pass the units when you actually compute the potential using `get_pairwise_zbl`. We could save the length unit and the energy unit as property.

Reply:
Could the ZBL class take the units as `__init__` parameters, do the conversion when instantiating itself and then store values in the right units?

Reply:
So, metatrain gives you the opportunity to train without any units. Do we error out in that case? And do we want to have a units module in metatrain, or can we perhaps recycle the one from ASE?

Reply:
> So, metatrain gives you the opportunity to train without any units. Do we error out in that case?

yes, I would error out in this case

> And do we want to have a units module in metatrain, or can we perhaps recycle the one from ASE?

I don't really mind, we can also use the code from metatensor: https://docs.metatensor.org/latest/atomistic/reference/models/index.html#metatensor.torch.atomistic.unit_conversion_factor